src/api/ingredients.js

```diff
@@ -0,0 +1,40 @@
+import axios from 'axios';
+
+const client = axios.create({
+  baseURL: 'http://localhost:8000/api/ingredients',
```

Comment:
What's the difference between this file and `api/recipes.js`? Is this used?

Reply:
Sorry, this file is not used. I deleted it.